pkg/plugins/client.go

```diff
@@ -137,6 +144,7 @@ func (c *Client) callWithRetry(serviceMethod string, data io.Reader, retry bool)
 		if err != nil {
 			return nil, err
 		}
+		req = req.WithContext(ctx)
```

Comment:
if this function retries with backoff then it shouldn't probably reuse the context as just waiting for a backoff will cause this context to close